packages/scheduler/src/components/Conflict/Conflict.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Conflict from './Conflict'
+import {shallow} from 'enzyme'
+import {IntlStub} from 'tocco-test-util'
+
+describe('scheduler', () => {
+  describe('components', () => {
+    describe('Conflict', () => {
+      it('should render return null if conflictstatus is undefined', () => {
+        const wrapper = shallow(<Conflict intl={IntlStub} />)
+        expect(wrapper.type()).to.be.null
+      })
+
+      it('should render return null if conflictstatus is undefined nones', () => {
```

Comment:
this should probably be 'should return null...' and '... conflictstatus is none'?